src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -108,19 +121,21 @@ def calculate_angle(self, p):
         angle = abs(theta1-theta2)
         return angle
 
-
-    def is_body_line_motion_downward(self, left_angle_with_yaxis, rigth_angle_with_yaxis):
+    def is_body_line_motion_downward(self, left_angle_with_yaxis, 
+                                            rigth_angle_with_yaxis, inx):
 
         test = False
-        l_angle = left_angle_with_yaxis and self._prev_left_angle_with_yaxis and left_angle_with_yaxis > self._prev_left_angle_with_yaxis
-        r_angle = rigth_angle_with_yaxis and self._prev_right_angle_with_yaxis and rigth_angle_with_yaxis > self._prev_right_angle_with_yaxis 
+
+        l_angle = left_angle_with_yaxis and self._prev_data[inx]['_prev_left_angle_with_yaxis']  \
```

Comment:
@bhavikapanara it seems like you are distracted and not thinking through comments. What is the purpose of defining constants if we don't actually use them as intended?

Reply:
my apologies @ivelin Please check it.